### ISSUE TITLE
Current State of cDock

### ISSUE THREAD

**Issue by dockdude** [w0lfschild/cDock2#200]
What's happening? No updates or announcement.

I am confused by the versioning and current state of support/development.
I am a paid customer and currently running 1.20 (cDock 2 or 3?) Hasn't been update past 6 months.
There are quite a few reported issues, and the last communication from the developer was Nov 2018, is this project abandoned?

Also, there are different versions in different places.
https://sourceforge.net/projects/cdock/files/cDock/ latest version here is 1.04
https://www.macenhance.com/cdock.html latest version or last change log is 1.1.3

If this project is still active, can we just name it cDock 3.x?

Just wondering...

Dockdude.

**Comment by w0lfschild** [comment]
Apologies for lack of communication recently.

Current version: cDock 3 (version # 1.2.0)

Is the project abandoned: No

Why is sourceforge old: I stopped using source forge a long while ago when they had some bad press about adding stuff to downloads without the users consent and taking over old repositories.

Issues: I've taken a break for a while and probably won't have much free time to look into issues until summer time rolls around for me. I know there are a few fixes and new stuff I have already worked on that I did not get the chance to publish before I've dropped out of the scene. Overall I'm looking to put out a new version of mySIMBL that is a significant overhaul rebranded to MacForge. That will also be effecting cDock as I'll switch over to using a different injection method for the plug-in most likely.

Hopefully that clears some stuff up. Sorry if formatting is bad this was sent from my phone.
